vm/gasCost.go

```diff
@@ -28,6 +28,7 @@ type MetaChainSystemSCsCost struct {
 	DelegateVote          uint64
 	RevokeVote            uint64
 	CloseProposal         uint64
+	ClearProposal         uint64
```

Comment:
you did not define the cost in the gas schedules toml files. Also, the tests fail because of this empty gas cost.

Reply:
fixed